tests/get-coverage.yml

```diff
@@ -0,0 +1,73 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: Prepare for coverage extraction
+  hosts: all
+  tasks:
+    - name: set facts
+      set_fact:
+        coverage_module: network_connections
+        coverage: /root/.local/bin/coverage
+    - name: set more facts
+      set_fact:
+        coverage_file: ansible-coverage-{{ coverage_module }}-{{ test_playbook|replace('.yml', '') }}
+
+    - name: set more facts
+      set_fact:
+        destdir: "remote_coverage/{{ test_playbook }}"
+      when: destdir is undefined
+
```

Comment:
ditto. A variable set in `vars:` will be overriden by `-e `, so `when:` should not be needed.
http://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable

Reply:
I removed it.